aptos-move/framework/aptos-stdlib/sources/cryptography/algebra_bls12381.move

```diff
@@ -1,11 +1,214 @@
 /// This module defines marker types, constants and test cases for working with BLS12-381 curves
 /// using generic API defined in `algebra.move`.
 ///
-/// Currently supported BLS12-381 structures include field `Fr`.
+/// See https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-pairing-friendly-curves-11#name-bls-curves-for-the-128-bit-
```

Comment:
In a future PR, let's fully implement inline functions here that just forward the calls to their generic counterparts. 

This will serve as good documentation for how to use the generic `algebra` module.